src/linux/component.rs

```diff
@@ -89,9 +89,10 @@ pub struct Component {
     // 	Unit: millidegree Celsius
     // 	RW
     /// temp[1-*]_input file to read current temperature.
+    /// It may be absent but we don't continue if absent.
```

Comment:
Then please wrap it into an `Option`. ;)

Reply:
The behavior is that we abandon the `component` if this file don't exist and other exist for `temperature`.

Maybe the docstring is not precise enough.

Reply:
A next PR can remove this `input_file` or update it to `Option<PathBuf>` I guess so this PR can get merged. :laughing:

Reply:
I prefers doing it myself I understand your move. :ninja: go for the Option.

Reply:
That's done. :cake:.